fast64_internal/utility.py

```diff
@@ -1896,3 +1896,12 @@ def create_or_get_world(scene: Scene) -> World:
         WORLD_WARNING_COUNT = 0
         print(f'No world in this file, creating world named "Fast64".')
         return bpy.data.worlds.new("Fast64")
+
+
+def set_if_different(owner: object, prop: str, value):
+    if getattr(owner, prop) != value:
+        setattr(owner, prop, value)
+
+
+def set_prop_if_in_data(owner: object, prop_name: str, data: dict, data_name: str):
+    set_if_different(owner, prop_name, data.get(data_name, getattr(owner, prop_name)))
```

Comment:
I think it would be less confusing with more verbosity
```py
    if data_name in data:
        set_if_different(owner, prop_name, data[data_name])
```